### ISSUE TITLE
Use new my wallet service for everything

### ISSUE THREAD

**Issue by anver** [blockchain/service-my-wallet-v3#102]
Service v0.20.1, Node v4.2.6

I would like to use my wallet server for all my sending and receiving bitcoins on my payment platform.
I've integrated blockchain.info wallet api service as a plugin in wordpress. I would like to use the new xpub key to generate addresses and pass the callback url as mentioned in the documentation. I know all that can be done using the wallet service but I don't understand how to implement sending and receiving from the same wallet using the new wallet service, right now this is how my system works.

**Step 1 :**
Setup a callback url in my old wallet settings ( not the beta since it doesn't have any option to set callback urls ) which points to my wordpress installed url something like http://mywpsite.com/wp-admin/admin-ajax.php?action=process_blockchain&secret=secretcode

**Step 2**
When the User clicks the payment button I generate the new address using the my wallet service setting the address label as my order number so that i can track my order. All newly generated addresses are stored in case if the user clicks the button multiple times. I don't use the new xpub key method to generate the addresses.

**Step 3**
When the payment is done by the user the ipn url that's set in my blockchain wallet settings fires which calls my script on my server. The script gets the received address and checks the label of the generated received address using the ( List Addresses ) my wallet v3 service.

**Step 4**
There are multiple checks to make sure that the payment made is valid etc then after all the checks are complete the user gets their product or memberships whatever...

**Step 5**
Apart from the product or membership delivery I have an affiliate system setup so it calculates different compensation plans for the uplines who promoted the product, it's like when a sale is made multiple people gets paid instantly using bitcoins from the same received address.

**Step 6**
After every compensation payouts are done the remaining bitcoins in the received address is transferred to an other account for the company.

The above steps are all working fine and I'm still using the old wallet and not the beta wallet, the moment the money comes in it splits the bitcoins and pays out instantly to all the users who share the revenue I know how to generate new address and set a dynamic callback url with the order id attached to that url etc using the new api and xpub key. 

The thing is I can receive funds without any issue but where I'm struck is I need to send the compensation payouts to the users who shares the revenue from the same received address. Is there any way I can use the my-wallet-v3 service access my beta wallet to send funds to the people from the same received address ?

**Comment by jtormey** [comment]
Sounds like you have a pretty good setup!

Regarding your question about spending, I'm not 100% sure what the issue is. You should be able to send funds from any wallet (provided you have the password) using this api module. Is it that you want to send bitcoin from a specific address within an account?
